dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -888,8 +887,8 @@ private boolean checkAndImportProcessDefinition(User loginUser, String currentPr
 
         //create process definition
         Integer processDefinitionId =
-                Objects.isNull(createProcessResult.get(PROCESSDEFINITIONID))
-                        ? null : Integer.parseInt(createProcessResult.get(PROCESSDEFINITIONID).toString());
+                Objects.isNull(createProcessResult.get(Constants.DATA_LIST))
```

Comment:
I think it would be better to keep the original one. PROCESSDEFINITIONID clearly expresses the field key to be obtained here.